torchvision/transforms/functional.py

```diff
@@ -478,7 +478,7 @@ def resize(
         size = [size]
     output_size = _compute_resized_output_size((image_height, image_width), size, max_size)
 
-    if (image_height, image_width) == output_size:
+    if [image_height, image_width] == output_size:
```

Comment:
Just explaining: this has to be a list here, since we return that from `_compute_resized_output_size`:

https://github.com/pytorch/vision/blob/b78d98bb152ffb9c0c0f5365f59f475c70b1784e/torchvision/transforms/functional.py#L388

Reply:
@pmeier do you remember why we couldn't make `_compute_resized_output_size -> Tuple[int, int]`, most probably JIT but IMO it would be simplier if `_compute_resized_output_size` returns a tuple instead of a list especially as we do not intend to append or modify the output size...

Reply:
I don't sorry. I agree that having a tuple there would be better. But this probably entails quite a few changes due to JIT. Not sure if its worth it. If you want to send a PR, happy to review.